reference-implementation/lib/writable-stream.js

```diff
@@ -713,10 +713,34 @@ class WritableStreamDefaultController {
     Promise.resolve(startResult).then(
       () => {
         controller._started = true;
-        WritableStreamDefaultControllerAdvanceQueueIfNeeded(controller);
+        const abortRequest = stream._pendingAbortRequest;
```

Comment:
Is there any way to deduplicate this with the code in WritableStreamFinishPendingWrite ? They seem to serve rather similar purposes...